app/index.js

```diff
@@ -81,6 +81,7 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'license',
         message: 'Which license do you want to use?',
         default: this.options.defaultLicense,
+        when: this.options.license == null || licenses.find(x => x.value === this.options.license) === undefined,
```

Comment:
I think the comment was about the `== null` like the other prompt. For this prompt it doesn't matter, but for the other ones it's important to allow passing empty values. I'm in favor of keeping the equality checks consistent in this case.